--- kaffe-1.1.5/configure.ac.orig	2005-04-09 11:07:23.000000000 +0000
+++ kaffe-1.1.5/configure.ac	2005-04-09 11:09:07.000000000 +0000
@@ -2149,7 +2149,14 @@
 toolslibdir=$prefix/lib
 AC_SUBST(toolslibdir)
 
-jredir=$prefix/jre
+AC_ARG_WITH([jredir],
+	    AS_HELP_STRING([--with-jredir=DIR],
+			   [The directory to install JRE in]))
+if test x"$with_jredir" = x"" ; then
+	jredir=$prefix/jre
+else
+	jredir=$with_jredir
+fi
 AC_SUBST(jredir)
 
 jrebindir=$jredir/bin
--- kaffe-1.1.7/kaffe/scripts/compat/javac.in.orig	2005-04-10 13:33:59.000000000 +0200
+++ kaffe-1.1.7/kaffe/scripts/compat/javac.in	2005-04-10 13:34:11.000000000 +0200
@@ -1,4 +1,4 @@
 #! /bin/sh
 # We use the jikes compiler detected by configure
-@JIKESPROG@ -bootclasspath  @prefix@/jre/lib/glibj.zip ${1+"$@"}
+@JIKESPROG@ -bootclasspath  @jredir@/lib/glibj.zip ${1+"$@"}
 exit $?
