--- kaffe-1.0.6/config/arm/trampolines.c.orig	1999-10-16 22:52:39.000000000 +0200
+++ kaffe-1.0.6/config/arm/trampolines.c	2004-03-31 21:07:55.297284144 +0200
@@ -35,13 +35,13 @@
 
 asm(
 	START_ASM_FUNC() C_FUNC_NAME(arm_do_fixup_trampoline) "\n"
-C_FUNC_NAME(arm_do_fixup_trampoline) ":				\n
-	stmdb	sp!,{r0,r1,r2,r3,lr}\n
-	mov	r0,ip\n
-	bl	" C_FUNC_NAME(soft_fixup_trampoline) "		\n
-	mov	ip, r0\n
-	ldmia	sp!,{r0,r1,r2,r3,lr}\n
-	mov	pc, ip\n
+C_FUNC_NAME(arm_do_fixup_trampoline) ":				\n\
+	stmdb	sp!,{r0,r1,r2,r3,lr}\n\
+	mov	r0,ip\n\
+	bl	" C_FUNC_NAME(soft_fixup_trampoline) "		\n\
+	mov	ip, r0\n\
+	ldmia	sp!,{r0,r1,r2,r3,lr}\n\
+	mov	pc, ip\n\
 "
 	END_ASM_FUNC()
 );
@@ -50,13 +50,13 @@
 
 asm(
 	START_ASM_FUNC() C_FUNC_NAME(arm_do_fixup_trampoline) "\n"
-C_FUNC_NAME(arm_do_fixup_trampoline) ":				\n
-	stmdb	sp!,{r0,r1,r2,r3,lr}\n
-	mov	r0,ip\n
-	bl	" C_FUNC_NAME(soft_fixup_trampoline) " (PLT)	\n
-	mov	ip, r0\n
-	ldmia	sp!,{r0,r1,r2,r3,lr}\n
-	mov	pc, ip\n
+C_FUNC_NAME(arm_do_fixup_trampoline) ":				\n\
+	stmdb	sp!,{r0,r1,r2,r3,lr}\n\
+	mov	r0,ip\n\
+	bl	" C_FUNC_NAME(soft_fixup_trampoline) " (PLT)	\n\
+	mov	ip, r0\n\
+	ldmia	sp!,{r0,r1,r2,r3,lr}\n\
+	mov	pc, ip\n\
 "
 	END_ASM_FUNC()
 );
--- kaffe-1.0.6/config/i386/trampolines.c.orig	1999-12-06 07:47:17.000000000 +0100
+++ kaffe-1.0.6/config/i386/trampolines.c	2004-03-31 21:06:54.007601592 +0200
@@ -33,9 +33,9 @@
 
 asm(
 	START_ASM_FUNC() C_FUNC_NAME(i386_do_fixup_trampoline) "\n"
-C_FUNC_NAME(i386_do_fixup_trampoline) ":			\n
-	call	" C_FUNC_NAME(soft_fixup_trampoline) "		\n
-	popl	%ecx						\n
+C_FUNC_NAME(i386_do_fixup_trampoline) ":			\n\
+	call	" C_FUNC_NAME(soft_fixup_trampoline) "		\n\
+	popl	%ecx						\n\
 	jmp	*%eax"
 	END_ASM_FUNC()
 );
@@ -48,16 +48,16 @@
  */
 asm(
 	START_ASM_FUNC() C_FUNC_NAME(__kaffe_i386_gcj_fixup) "\n"
-C_FUNC_NAME(__kaffe_i386_gcj_fixup) ":
-	mov	(%esp), %eax	# get return address
-	add	-4(%eax), %eax	# add jump relative offset from previous instr.
-				# this points at at jmp *$off(%ebx) instr.
-	mov	2(%eax), %eax	# extract 'off'
-	add	%ebx, %eax	# compute $off(%ebx)
-	pushl	%eax		# pass as first argument
-	call	" C_FUNC_NAME(gcj_fixup_trampoline) " # returns target
-	addl	$4, %esp	# remove argument
-	jmp	*%eax		# jump to target
+C_FUNC_NAME(__kaffe_i386_gcj_fixup) ":\n\
+	mov	(%esp), %eax	# get return address\n\
+	add	-4(%eax), %eax	# add jump relative offset from previous instr.\n\
+				# this points at at jmp *$off(%ebx) instr.\n\
+	mov	2(%eax), %eax	# extract 'off'\n\
+	add	%ebx, %eax	# compute $off(%ebx)\n\
+	pushl	%eax		# pass as first argument\n\
+	call	" C_FUNC_NAME(gcj_fixup_trampoline) " # returns target\n\
+	addl	$4, %esp	# remove argument\n\
+	jmp	*%eax		# jump to target\n\
 "
 	END_ASM_FUNC()
 );
--- kaffe-1.0.6/config/m68k/trampolines.c.orig	1999-12-18 23:51:10.000000000 +0100
+++ kaffe-1.0.6/config/m68k/trampolines.c	2004-03-31 21:08:48.133251856 +0200
@@ -40,10 +40,10 @@
 
 asm (
         START_ASM_FUNC() C_FUNC_NAME(m68k_do_fixup_trampoline) "\n"
-C_FUNC_NAME(m68k_do_fixup_trampoline) ":
-        jbsr    " C_FUNC_NAME(soft_fixup_trampoline) "
-	addqw	#4,%sp
-	movel	%d0,%a0
+C_FUNC_NAME(m68k_do_fixup_trampoline) ":\n\
+        jbsr    " C_FUNC_NAME(soft_fixup_trampoline) "\n\
+	addqw	#4,%sp\n\
+	movel	%d0,%a0\n\
         jmp     %a0@"
         END_ASM_FUNC()
 );
@@ -52,10 +52,10 @@
 
 asm (
         START_ASM_FUNC() C_FUNC_NAME(m68k_do_fixup_trampoline) "\n"
-C_FUNC_NAME(m68k_do_fixup_trampoline) ":
-        jbsr    " C_FUNC_NAME(soft_fixup_trampoline) "
-        addqw   #4,sp
-        movel   d0,a0
+C_FUNC_NAME(m68k_do_fixup_trampoline) ":\n\
+        jbsr    " C_FUNC_NAME(soft_fixup_trampoline) "\n\
+        addqw   #4,sp\n\
+        movel   d0,a0\n\
         jmp     a0@"
         END_ASM_FUNC()
 );
--- kaffe-1.0.6/config/sparc/linux/md.c.orig	1998-10-01 22:55:08.000000000 +0200
+++ kaffe-1.0.6/config/sparc/linux/md.c	2004-03-31 21:09:44.650659904 +0200
@@ -25,15 +25,15 @@
 #define	GET_SP(E)	(((void**)(E))[SP_OFFSET])
 
 extern int __speciallongjmp(void*);
-asm("
-__speciallongjmp:
-	mov %o0,%g1
-	ta 3
-	ld [%g1],%sp
-	ld [%g1+4],%fp
-	ld [%g1+8],%g1
-	jmp %g1+8
-	mov 1,%o0
+asm("\n\
+__speciallongjmp:\n\
+	mov %o0,%g1\n\
+	ta 3\n\
+	ld [%g1],%sp\n\
+	ld [%g1+4],%fp\n\
+	ld [%g1+8],%g1\n\
+	jmp %g1+8\n\
+	mov 1,%o0\n\
 ");
 
 void
--- kaffe-1.0.6/config/sparc/trampolines.c.orig	1999-11-04 21:48:40.000000000 +0100
+++ kaffe-1.0.6/config/sparc/trampolines.c	2004-03-31 21:10:03.733758832 +0200
@@ -33,13 +33,13 @@
 
 asm(
 	START_ASM_FUNC() C_FUNC_NAME(sparc_do_fixup_trampoline) "\n"
-C_FUNC_NAME(sparc_do_fixup_trampoline) ":			\n
-	save	%sp,-64,%sp					\n
-	ld	[%i7+8],%o0					\n
-	ld	[%i7+12],%o1					\n
-	call	" C_FUNC_NAME(soft_fixup_trampoline) "		\n
-	mov	%g1,%i7						\n
-	jmp	%o0						\n
+C_FUNC_NAME(sparc_do_fixup_trampoline) ":			\n\
+	save	%sp,-64,%sp					\n\
+	ld	[%i7+8],%o0					\n\
+	ld	[%i7+12],%o1					\n\
+	call	" C_FUNC_NAME(soft_fixup_trampoline) "		\n\
+	mov	%g1,%i7						\n\
+	jmp	%o0						\n\
 	restore"
 	END_ASM_FUNC()
 );
