diff -Nru kaffe-1.0.5/developers/dumpClass.pl kaffe-1.0.5.new/developers/dumpClass.pl
--- kaffe-1.0.5/developers/dumpClass.pl	Sat Jan  8 00:33:48 2000
+++ kaffe-1.0.5.new/developers/dumpClass.pl	Sat Jan  8 00:32:19 2000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/perl
 #
 # Dump a java .class file to stdout
 #
@@ -12,20 +12,6 @@
 #
 # Written by Patrick Tullmann <tullmann@cs.utah.edu>
 #
-
-if [ -x /usr/bin/perl ]; then
-	PERL=/usr/bin/perl
-else
-	if [ -x /usr/local/bin/perl ]; then
-		PERL=/usr/local/bin/perl
-	fi
-fi
-
-PERL5INC=`echo "$0" | sed 's,/[^/]*$,,'`  # very portable
-$PERL -I $PERL5INC -x $0 ${1+"$@"}
-exit
-
-#!perl
 
 use JavaClass;
 
diff -Nru kaffe-1.0.5/developers/utf8munge.pl kaffe-1.0.5.new/developers/utf8munge.pl
--- kaffe-1.0.5/developers/utf8munge.pl	Mon Oct 18 07:23:54 1999
+++ kaffe-1.0.5.new/developers/utf8munge.pl	Sat Jan  8 00:33:31 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Munge the Utf8 constants in a .class file.
 #
