--- kaffe-1.0b4/libraries/javalib/java/lang/String.java.orig	Sun Mar 21 05:38:45 1999
+++ kaffe-1.0b4/libraries/javalib/java/lang/String.java	Tue Feb 29 17:11:33 2000
@@ -211,7 +211,7 @@
 private byte[] getBytes( CharToByteConverter encoding) {
 	ByteArrayOutputStream out = new ByteArrayOutputStream( value.length);
 
-	byte[] buf = new byte[512];
+	byte[] buf = new byte[value.length*7];
 	int buflen = encoding.convert( value, offset, count, buf, 0, buf.length);
 	while (buflen > 0) {
 		out.write(buf, 0, buflen);
