--- kaffe-1.0.6/kaffe/kaffevm/debug.h.jlong	Fri Jun 16 07:18:26 2000
+++ kaffe-1.0.6/kaffe/kaffevm/debug.h	Tue Aug  8 19:58:24 2000
@@ -27,6 +27,7 @@
  * is include-once).
  */
 #include <stdio.h>
+#include "jtypes.h"
 
 /*
  * Available debugging options.
@@ -202,7 +203,7 @@
 #endif /* defined(NDEBUG) || !defined(DEBUG) */
 
 /* Set the debugging mask to use. (give the mask) */
-void dbgSetMask(long long mask);
+void dbgSetMask(jlong mask);
 
 /* 
  * Set the debugging mask to use. (give a string, useful for
