diff -Nur kaffe-1.0.6.orig/test/regression/Makefile.am kaffe-1.0.6/test/regression/Makefile.am
--- kaffe-1.0.6.orig/test/regression/Makefile.am	Mon Jul 24 22:44:27 2000
+++ kaffe-1.0.6/test/regression/Makefile.am	Fri Oct 19 23:53:05 2001
@@ -11,11 +11,8 @@
 TESTS_ENVIRONMENT = $(SETUP_ENVIRONMENT) $(SHELL) TestScript
 
 ALWAYS_TESTS = \
-## Simple health check
 	HelloWorldApp.class.save \
-## Now test the compiler
 	HelloWorldApp.java \
-## Test ints, longs, floats and doubles
 	TestIntLong.java \
 	TestFloatDouble.java \
 	DoubleNeg.java \
@@ -24,23 +21,18 @@
 	ModuloTest.java \
 	LongNeg.java \
 	FPUStack.java \
-## Test strings
 	Str.java \
 	Str2.java \
 	InternHog.java \
-## Test exceptions
 	IndexTest.java \
 	StackDump.java \
-## Test threads
 	tname.java \
 	ttest.java \
 	ThreadInterrupt.java \
 	ThreadState.java \
 	UncaughtException.java \
 	IllegalWait.java \
-## Test that preemption works---may not be supported by all threading systems
-	Preempt.java 		\
-## Misc tests
+	Preempt.java \
 	ReflectInterfaces.java \
 	TestSerializable.java \
 	TestSerializable2.java \
@@ -84,11 +76,8 @@
 	LineNumberReaderTest.java \
 	BufferedReaderTest.java \
 	ReaderReadVoidTest.java \
-## simple test for Method.invoke
 	ReflectInvoke.java \
-## simple test for InvocationTargetExceptions
 	InvTarExcTest.java \
-## tests for ClassLoader
 	CLTest.java \
 	CLTestConc.java \
 	CLTestJLock.java \
@@ -117,20 +106,14 @@
 	HashtableTest1.java \
 	ReflectMultiArray.java \
 	ExecTest.java \
-## Test catching our own death
 	CatchDeath.java \
-## Test various ways of stopping threads
 	ThreadStop.java \
 	DeadThread.java \
 	tthrd1.java \
-## Test interrupting I/O
-## This is kind of tricky, and Sun doesn't support it.
 	SoInterrupt.java \
 	sysdepCallMethod.java \
-## Test for package java.util.zip
 	DosTimeVerify.java \
 	ZipVerify.java \
-## This one is skipped unless you --enable-debug.
 	TestNative.java
 
 TEST_Overflow = Overflow.java
--- kaffe-1.0.6.orig/kaffe/kaffevm/Makefile.am	Tue May 23 19:55:35 2000
+++ kaffe-1.0.6/kaffe/kaffevm/Makefile.am	Sat Oct 20 00:00:13 2001
@@ -29,18 +29,19 @@
 @cpu_os_frag@
 @threads_frag@
 
+libkaffevm_la_LDFLAGS = $(KVMLIBFLAGS) -export-dynamic -release $(KVER)
 if HAVE_GCJ_SUPPORT
-libgcj = gcj/libgcj.la
+libkaffevm_la_DEPENDENCIES = \
+	$(LIBLTDL) \
+	$(ENGINE_NAME)/libengine.la \
+	gcj/libgcj.la \
+	systems/$(THREAD_SYSTEM)/libkthread.la
 else
-libgcj = #
-endif
-
-libkaffevm_la_LDFLAGS = $(KVMLIBFLAGS) -export-dynamic -release $(KVER)
 libkaffevm_la_DEPENDENCIES = \
 	$(LIBLTDL) \
 	$(ENGINE_NAME)/libengine.la \
-	$(libgcj) \
 	systems/$(THREAD_SYSTEM)/libkthread.la
+endif
 libkaffevm_la_LIBADD = $(libkaffevm_la_DEPENDENCIES) $(VM_LIBS) $(M_LIBS)
 
 libkaffevm_la_GEN_SRCS = \
