--- kaffe-1.0.6/configure.in~	Mon Jul 24 21:43:44 2000
+++ kaffe-1.0.6/configure.in	Sat Apr 14 01:50:22 2001
@@ -676,7 +676,7 @@
 dnl -------------------------------------------------------------------------
 
 AC_HEADER_STDC
-AC_CHECK_HEADERS(fcntl.h sys/time.h unistd.h stdlib.h sys/socket.h winnt.h wintypes.h wtypes.h winbase.h windows.h winsock.h dlfcn.h sys/ioctl.h sys/filio.h malloc.h alloca.h sys/param.h sys/utsname.h pwd.h asm/sigcontext.h sigcontext.h asm/signal.h signal.h mach-o/rld.h sys/types.h sys/stat.h string.h sys/select.h poll.h sys/poll.h memory.h bsd/libc.h dl.h features.h sys/wait.h zlib.h netinet/tcp.h netinet/in_systm.h netinet/in.h netdb.h jpeglib.h png.h gmp.h utime.h kernel/OS.h)
+AC_CHECK_HEADERS(fcntl.h time.h sys/time.h unistd.h stdlib.h sys/socket.h winnt.h wintypes.h wtypes.h winbase.h windows.h winsock.h dlfcn.h sys/ioctl.h sys/filio.h malloc.h alloca.h sys/param.h sys/utsname.h pwd.h asm/sigcontext.h sigcontext.h asm/signal.h signal.h mach-o/rld.h sys/types.h sys/stat.h string.h sys/select.h poll.h sys/poll.h memory.h bsd/libc.h dl.h features.h sys/wait.h zlib.h netinet/tcp.h netinet/in_systm.h netinet/in.h netdb.h jpeglib.h png.h gmp.h utime.h kernel/OS.h)
 AC_HEADER_DIRENT
 
 dnl We need both getrusage and the header containing its decls.
--- kaffe-1.0.6/config/config.h.in~	Sun Feb 13 02:07:39 2000
+++ kaffe-1.0.6/config/config.h.in	Sat Apr 14 01:52:34 2001
@@ -367,6 +367,9 @@
 /* Define if you have the <sys/stat.h> header file.  */
 #undef HAVE_SYS_STAT_H
 
+/* Define if you have the <time.h> header file.  */
+#undef HAVE_TIME_H
+
 /* Define if you have the <sys/time.h> header file.  */
 #undef HAVE_SYS_TIME_H
 
